iphone/Maps/Classes/CustomAlert/MWMOsmReauthAlert.mm

```diff
@@ -0,0 +1,69 @@
+#import "MWMAlertViewController.h"
+#import "MWMOsmReauthAlert.h"
+#import "MWMAuthorizationCommon.h"
+
+#include "editor/osm_auth.hpp"
+
+static NSString * const kMap2OsmLoginSegue = @"Map2OsmLogin";
+
+@implementation MWMOsmReauthAlert
+
++ (instancetype)alert
+{
+  MWMOsmReauthAlert * alert =
+      [NSBundle.mainBundle loadNibNamed:[self className] owner:nil options:nil].firstObject;
```

Comment:
No need to split these lines in XCode. Longer lines are easier to read in some cases.
```suggestion
  MWMOsmReauthAlert * alert = [NSBundle.mainBundle loadNibNamed:[self className] owner:nil options:nil].firstObject;
```